powershell/plugins/plugin-create-inline-properties.ts

```diff
@@ -363,16 +364,12 @@ async function createVirtuals(state: State): Promise<PwshModel> {
  
     Individual models can change the $THRESHOLD for generate
   */
-  // dolauli this plugin is used to expand the property in an object.
-  // dolauli inline-threshold could be set in readme.md
-  // skip-for-time-being
-  const threshold = await state.getValue('inlining-threshold', 24);
+  // const threshold = await state.getValue('inlining-threshold', 24);
+  const threshold = 24;
```

Comment:
Any special reason to fix the threshold to 24?

Reply:
revert the change. Thanks